src/_data/audioworklet_data.yaml

```diff
@@ -52,6 +52,11 @@
         This example demonstrates how this library can be used as a channel
         between a worker thread and AudioWorklet to send audio data.
       href: free-queue/examples/simple-passthrough/
+    - title: Hello WebAssembly
+      description: Example of using FreeQueue with WebAssembly.
```

Comment:
An example demonstrates the usage of FreeQueue with WebAssembly. It includes an external C/C++ MP3 decoder library to load and play music with FreeQueue on top of AudioWorklet.